katdal/visdatav4.py

```diff
@@ -451,7 +456,8 @@ def _register_standard_cal_streams(self):
             for prefix in reversed(attrs.prefixes):
                 l2_attrs = l2_attrs.view(prefix + l2_streams[0])
             l2_freqs = add_applycal_sensors(self.sensor, l2_attrs, freqs,
-                                            cal_stream='l2', cal_substreams=l2_streams)
+                                            cal_stream='l2', cal_substreams=l2_streams,
```

Comment:
Adding `gaincal_fluxes` to l2 gains is a worrying as you should not be flux scaling the self-calibrated gains.

Reply:
Removed this for clarity (would only have happened if l2 had `G` and not `GPHASE`).